### ISSUE TITLE
Configuration for swiftlint

### ISSUE THREAD

**Issue by yasirmturk** [raywenderlich/swift-style-guide#305]
Is there any swiftlint.yml configuration file available for these style guidelines?

**Comment by christiankm** [comment]
Most standard rules in Swiftlint follow this guideline. But im interested as well.
We've made som custom rules to adhere to the guidelines, and our own exceptions.

**Comment by yasirmturk** [comment]
yeah also with those custom rules, also if there is a comparison of rules of all guidelines available somewhere that would be handy.
google vs linkedin vs swiftlint vs raywenderlich
